src/lib/dai.c

```diff
@@ -218,6 +221,8 @@ static int sof_dai_type_to_zephyr(uint32_t type)
 	case SOF_DAI_AMD_HS_VIRTUAL:
 	case SOF_DAI_AMD_SW_AUDIO:
 		return -ENOTSUP;
+	case SOF_DAI_VIRTUAL:
+		return DAI_VIRTUAL;
 	default:
```

Comment:
Whill this patch compile? Looks there is no `DAI_VIRTUAL` definition yet. So each patch when applied should compile withtout the nxt patches.

So we don't break bisectability,